controllers/auth.go

```diff
@@ -125,8 +123,8 @@ func GoogleCallback(ctx *gin.Context, db *bun.DB) {
         ctx.JSON(http.StatusInternalServerError, gin.H{"message": "error"})
         return
     }
-
-    ctx.SetCookie("token", token, tokenExpiration, "/", domain, true, true)
+    fmt.Println("config token expiration", config.TokenExpiration)
```

Comment:
do we need to print this

Reply:
No doing for testing the app  will remove after fixing the failing test